Interfaces/API/Dirac.py

```diff
@@ -993,6 +993,46 @@ def __printOutput(fd=None, message=''):
     else:
       print(message)
 
+  #############################################################################
+  def listCatalogDir(self, directoryLFN, printOutput=False):
+    """lists the contents of a directory in the DFC
+    example usage: dirac.listCatalogDir("/lz/data/test", printOutput=True)
+    param directoryLFN: LFN of the directory to be listed
```

Comment:
```suggestion

    :param directoryLFN: LFN of the directory to be listed
```
(note the empty line before the first `:param....:` the other lines also need a leading colon

Reply:
And I was so proud that I passed the pylint test.

Reply:
I forgot the ':', but otherwise followed the example in def getReplicas (the next function in the file), which uses boolean (and not bool) and uses two different lines for param and type. Which version would you like @fstagni ?

Reply:
You need to do the :eye: lint test https://dirac--5201.org.readthedocs.build/en/5201/CodeDocumentation/Interfaces/API/Dirac.html?highlight=listcatalogdir#DIRAC.Interfaces.API.Dirac.Dirac.listCatalogDir

:grin:

Reply:
![image](https://user-images.githubusercontent.com/7568639/122250964-0521a380-cec2-11eb-8a8f-006d7a29ef67.png)
I might have to stay fluffy..... 
How was I meant o find the link above ?

Reply:
The type in python is `bool`, not `boolean` or `Boolean`, so if you use bool it  gets linked to the doc for bool
E.g.: https://dirac--5201.org.readthedocs.build/en/5201/CodeDocumentation/DataManagementSystem/Client/DataManager.html?#DIRAC.DataManagementSystem.Client.DataManager.DataManager.getReplica

Reply:
Looks fine to me now.